install_linux.py

```
@@ -175,10 +175,30 @@ def install_frontend_deps(self):
             return True
 
         if not shutil.which("npm"):
-            print(" ⚠️ npm not found. Please install Node.js 18+ (e.g., via nvm or system package).")
+            print(" ⚠️ npm not found. Please install Node.js 20.19+ or 22 LTS (e.g., via nvm or system package).")
             self.logger.warning("npm not found. Frontend setup skipped.")
             return False
 
+        # Check that Node.js meets the minimum version requirement (>=20.19.0)
+        try:
+            node_result = subprocess.run(["node", "--version"], capture_output=True, text=True)
```

Ruff (0.15.6)

[error] Starting a process with a partial executable path

(S607)